Unit test must be updated:
```
 [ERROR] acl_request_by_config_server(com.yahoo.vespa.hosted.provision.restapi.NodesV2ApiTest)  Time elapsed: 0.196 s  <<< FAILURE!
15:19:10 org.junit.ComparisonFailure: acl-config-server.json expected:<...y":"cfg1.yahoo.com"}[]]}> but was:<...y":"cfg1.yahoo.com"}[,{"port":51820,"trustedBy":"cfg1.yahoo.com"}]]}>
```

--- a/node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/AclProvisioningTest.java
+++ b/node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/AclProvisioningTest.java
@@ -109,7 +109,7 @@ public class AclProvisioningTest {
                            TrustedNode.of(configNodes)),
                    Set.of("10.2.3.0/24", "10.4.5.0/24"),
                    List.of(nodeAcl));
-        assertEquals(Set.of(22, 4443), nodeAcl.trustedPorts());
+        assertEquals(Set.of(22, 4443, 51820), nodeAcl.trustedPorts());
     }
 
     @Test

--- a/node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/responses/acl-config-server.json
+++ b/node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/responses/acl-config-server.json
@@ -274,6 +274,10 @@
     {
       "port": 4443,
       "trustedBy": "cfg1.yahoo.com"
+    },
+    {
+      "port":51820,
+      "trustedBy":"cfg1.yahoo.com"
     }
   ]
 }

